content/components/action-menu.mdx

```diff
@@ -102,89 +132,292 @@ Action menu is used for actions, navigation, to display secondary options, or si
+  </Dont>
+</DoDontContainer>
+
+Especially when providing single or multi select items that rely on a crossmark icon should be avoided as they could be confused with the item's checkmark.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/232508225-59f21afe-1613-4ad0-8088-a8f2f6eb2ba9.png"
+    />
+    <Caption>Avoid the usage of icons if the icon conflicts with the selected states checkmark.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/232508233-78d0460d-c891-42be-929d-ff0465313805.png"
+    />
+    <Caption>Don't use crossmark icons for single or multi select items.</Caption>
   </Dont>
 </DoDontContainer>
 
+### Dividers
+
+To enhance the readability of menus that contain numerous item descriptions, it is recommended to incorporate dividers. This can effectively prevent the menu from becoming overwhelming.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/234297298-76d14987-2ecb-4165-9f5d-ada0fd1c7e17.png"
+    />
+    <Caption>Introduce dividers for items with heavy descriptions to improve readability.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/234297304-0cfdd597-fc79-4394-bb74-b37ec1276bd6.png"
+    />
+    <Caption>Don't forget to add dividers when the menu content starts to look heavy.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Trailing visuals
+
+Don't use trailing visuals to add additional interactions instead keep them reserved for submenu indicators or keyboard shortcuts.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/234299628-3c8a8613-7a2f-4f4a-83da-14eab2f28032.png"
+    />
+    <Caption>Create a new section for additional functionality.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/234299623-927d9309-9991-496c-b707-b6f358583928.png"
+    />
+    <Caption>Don't store additional toggleable functionality in trailing visuals.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Multi-level
+
+A menu with multiple levels is a common UI component used in desktop and mobile applications. It allows users to access nested subitems without cluttering the screen.
+
+<img
+  width="960"
+  alt="Action menu opened with a label for the trigger (button), overlay (menu), and the internal action list."
+  src="https://user-images.githubusercontent.com/980622/232519615-d10c928d-5253-4c54-bc66-7fa1e0f85b08.png"
+/>
+
+It's important to understand that menus can't include additional form elements like inputs. That means if you want to provide a input that filters the subitems that you can't rely on this component.
```

Comment:
```suggestion
It's important to understand that menus can't include additional form elements like inputs. That means if you want to provide a input that filters the subitems that you can't rely on this component. Alternatively, you can use a [select panel](/components/selectpanel) or open a [dialog](/components/dialog) instead of a submenu.
```